cmc-ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/ClaimData.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.cmc.ccd.domain;
+
+import lombok.Builder;
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Data
+@Builder
+public class ClaimData {
+
+    private final String reason;
```

Comment:
Just out of curiosity -- is intellij smart enough with Lombok to not display errors for unassigned final field?

Reply:
yes, lambok has plugin for idea